pkgs/applications/audio/sayonara/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, cmake, qt5, zlib, taglib, pkgconfig, pcre, gst_all_1 }:
+
+let
+  version = "1.0.0-git5-20180115";
+in
+stdenv.mkDerivation {
+  name = "sayonara-player-${version}";
+
+  src = fetchurl {
+    url = "https://sayonara-player.com/sw/sayonara-player-${version}.tar.gz";
+    sha256 = "1fl7zplnrrvbv1xm4g348bpd46jj39jvbm808hyjjq92i64wqg37";
+  };
+
+  buildInputs = [ cmake qt5.qtbase qt5.qttools zlib taglib pkgconfig pcre gst_all_1.gstreamer gst_all_1.gst-plugins-base gst_all_1.gst-plugins-good gst_all_1.gst-plugins-ugly ];
```

Comment:
At least `cmake` and `pkgconfig` can be made into `nativeBuildInputs`

Reply:
Done..